src/pass/storage_rewrite.cc

```diff
@@ -564,8 +564,14 @@ class StoragePlanRewriter : public IRMutator {
           Expr combo_size;
           for (const Allocate* op : e->allocs) {
             Expr sz = arith::ComputeReduce<Mul>(op->extents, make_const(Int(32), 1));
+            auto nbits = op->type.bits() * op->type.lanes();
+            if (const auto* imm = sz.as<IntImm>()) {
+              if (imm->value > std::numeric_limits<int>::max() / nbits) {
+                sz = make_const(Int(64), imm->value);
+              }
+            }
```

Comment:
Suggest to add warning or todo comments here, indicate the return type might be different in different situation and the intention to do so.

Reply:
@yzhliu No problem. Thanks.